manifests/examples/azure/jessie.yml

```diff
@@ -23,7 +23,11 @@ volume:
     root:
       filesystem: ext4
       size: 10GiB
-packages: {}
+packages:
+  components:
+  - main
+  - contrib
+  - non-free
```

Comment:
That is still TBD.  But I do want bootstrap-vz to be *capable* of producing quality Azure images :)